src/D7/WatchdogTrait.php

```diff
@@ -73,4 +73,11 @@ public function checkWatchdog(AfterScenarioScope $scope) {
     }
   }
 
+  /**
```

Comment:
please move this to `FeatureContextD7.php` as this is used to test the trait itself.